### DISCUSSION TITLE
Artisan Events and buttons

### DISCUSSION THREAD

**Discussion by SaeedAlkatheri** [artisan-roaster-scope/artisan#880] [Q&A] [answered]
## Expected Behavior
I have a Roasting Probate machine 25Kg / and i am trying to control the airflow.
How can i fix this issue?
 is it possible if i control the airflow separately with the heat or the burner? 
Also do i need to add a commend or a code to do a communication between artisan and the machine for the airflow?

## Actual Behavior
I am not able to change any thing on the airflow sliders or the button there is no communication when it comes for the airflow. on the other hand the  heat or the burner is working fine and the burner is showing exact  number on the machine.

## Specifications

  - Artisan Version: 2..4.4
  - Artisan Build (number in brackets shown in the about box): 1fb3d349 / https://github.com/artisan-roaster-scope
/
[artisan](https://github.com/artisan-roaster-scope/artisan)
  - Platform (Mac/Windows/Linux + OS version): Windows 10
  - Connected devices or roasting machine: roasting Machine

Please attach your current Artisan settings file (as exported via menu Help >> Save Settings as *.aset) file.
Please attach any relevant Artisan *.alog profiles.
[artisan-settings.zip](https://github.com/artisan-roaster-scope/artisan/files/8629911/artisan-settings.zip)

_Note that you need either add a `.txt` extension or zip the files before uploading. Otherwise you will receive a "Not a supported file type" error on uploading._

**Comment by MAKOMO** [comment]
Why are you using the outdated Artisan version 2.4.4? We are now on 2.6.0!

There are different Probat roasting machines. What type of machine do you have? I assume this is not a new P Series III. Does it features the larger 7” display (Beckhof PLC) or the smaller one (Wago)?

The configuration you attached is for the machines with the smaller display (Wago). Those machines did never feature airflow control. For those machines with the larger display airflow and drum speed control was an option. For machines with the larger display you need to choose menu `Config >> Probat Probatone 7"`. That setup includes airflow control.

Please report if choosing this setup helps, but first update your Artisan!

**Reply by SaeedAlkatheri** [reply]
the machine name is Manual Probatone 25 type 2.
kindly check the attached photo.

the questions is are we able to control the airflow or the drum on this machine?

kindly advise.

Thank you.
![imgpsh_fullsize_anim-10](https://user-images.githubusercontent.com/104976220/167295995-70f12e2b-5cdb-4dca-90be-f56bf9ed41ae.png)

**Reply by MAKOMO** [reply]
Can you control drum speed and airflow from the machines touch screen? If the answer is yes, it can also controlled from within Artisan in case your machine has the software option booked (which is the case if Artisan can read data from the machine).

**Comment by MAKOMO** [comment, marked as answer]
Just looked at your settings. Your airflow control settings for both, the sliders and the buttons have been cleared. That's the reason why Artisan is not sending out any command to the machine on pressing those buttons or using that slider. Also the Event quantifier of the air event has been modified and now observes the BT instead of the airflow channel. Your setup has been modified in several respects.

I suggest to run again the setup for this machine to correct all of this (menu `Config >> Machine >> Probat 7"`).  During that setup you are asked for the machines IP address. In your current setup this is `169.254.231.229`. 

But please first update Artisan to the current version 2.6.0!

**Reply by MAKOMO** [reply]
Could you fix this?
